cumulus/client/service/src/lib.rs

```diff
@@ -458,6 +471,81 @@ where
 	Err("Stopping following imported blocks. Could not determine parachain target block".into())
 }
 
+/// Task for triggering backing group connections early.
+pub async fn collator_protocol_helper<Block: BlockT, Client, P>(
```

Comment:
Please move this code to the cumulus aura code

Reply:
done